CHANGELOG.md

```diff
@@ -3,14 +3,15 @@ All notable changes to this project 1.4.x per each release will be documented in
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
-## [v1.4.0.3] - TBA
+## [v1.4.0.3] - 2019-10-11
 
 ### Added
 - Docker Compose resources for Open Banking Berlin specification for Open Banking API Manager with Open Banking Key Manager profile
+- Docker Compose resources for Open Banking UK and Berlin specifications for Open Banking API Manager and Open Banking Key Manager with Open Banking Business profiles
 
 ### Changed
 - Separated DB scripts into different directories based on the specification
-
+- Set ulimits and max_connections of MySQL containers
 ### Added
```

Comment:
@kaumini combine the `Added` sections into one.

Reply:
Fixed with 226832a4f943bdf1d6a0ef36324c2c19a1f2dd9d